packages/@storefront/flux-capacitor/test/unit/core/reducers/data/navigations.ts

```diff
@@ -199,15 +199,90 @@ suite('navigations', ({ expect }) => {
       expect(reducer).to.eql(newState);
     });
 
-    it('should return state on SELECT_REFINEMENT if no navigationId and refinementIndex', () => {
+    it('should return state on SELECT_REFINEMENT if no navigationId', () => {
       const reducer = navigations(state, <any>{
         type: Actions.SELECT_REFINEMENT,
-        payload: {}
+        payload: { index: 1 },
+      });
+
+      expect(reducer).to.eql(state);
+    });
+
+    it('should return state on SELECT_REFINEMENT if no refinementIndex', () => {
+      const reducer = navigations(state, <any>{
+        type: Actions.SELECT_REFINEMENT,
+        payload: { navigationId: 'Section' },
+      });
+
+      expect(reducer).to.eql(state);
+    });
+
+    it('should add selected refinement state on SELECT_MULTIPLE_REFINEMENTS', () => {
+      const newState = {
+        ...state,
+        byId: {
+          Format,
+          Section: {
+            ...Section,
+            selected: [3, 0, 4, 9],
+          },
+          Department,
+        },
+      };
+
+      const reducer = navigations(state, {
+        type: Actions.SELECT_MULTIPLE_REFINEMENTS,
+        payload: {
+          navigationId: 'Section',
+          indices: [0, 4, 9]
+        }
+      });
+
+      expect(reducer).to.eql(newState);
+    });
+
+    it('should return state on SELECT_MULTIPLE_REFINEMENTS if no navigationId', () => {
+      const reducer = navigations(state, <any>{
+        type: Actions.SELECT_MULTIPLE_REFINEMENTS,
+        payload: { indices: [1, 2, 3] },
       });
 
       expect(reducer).to.eql(state);
     });
 
+    it('should return state on SELECT_MULTIPLE_REFINEMENTS if no indices', () => {
+      const reducer = navigations(state, <any>{
+        type: Actions.SELECT_MULTIPLE_REFINEMENTS,
+        payload: { navigationId: 'Section', indices: [] },
+      });
+
+      expect(reducer).to.eql(state);
+    });
+
+    it('should not add duplicate indices on SELECT_MULTIPLE_REFINEMENTS', () => {
+      const newState = {
+        ...state,
+        byId: {
+          Format,
+          Section: {
+            ...Section,
+            selected: [3, 0, 4, 9],
+          },
+          Department,
+        },
+      };
+
+      const reducer = navigations(state, {
+        type: Actions.SELECT_MULTIPLE_REFINEMENTS,
+        payload: {
+          navigationId: 'Section',
+          indices: [0, 4, 9, 3]
```

Comment:
Is the order significant?

Reply:
Yes order is significant because of the show property: show specifically determines which indices to show and what order they should show up in.

Reply:
Noted, thanks for this. Resolving.